runner/tests/operator/cmo_ch/v2_1_0/nucleo/nucleo_operator.py

```diff
@@ -0,0 +1,136 @@
+import os
+import json
+import logging
+
+from jinja2 import Template
+from collections import defaultdict
+
+from runner.operator.operator import Operator
+from runner.run.objects.run_creator_object import RunCreator
+from file_system.repository.file_repository import FileRepository
+
+
+logger = logging.getLogger(__name__)
+WORKDIR = os.path.dirname(os.path.abspath(__file__))
+
+METADATA_OUTPUT_FIELDS = [
+    "barcodeId",
+    "sampleName",
```

Comment:
Should we reference some of the metadata keys from `settings.py`

Reply:
Added a commit to address this